scripts/lib/setup-apt-repo

```diff
@@ -46,16 +46,22 @@ deb-src http://apt.postgresql.org/pub/repos/apt/ $release-pgdg main
 deb http://ppa.launchpad.net/groonga/ppa/ubuntu $release main
 deb-src http://ppa.launchpad.net/groonga/ppa/ubuntu $release main
 EOF
+    wget "https://packages.groonga.org/ubuntu/groonga-apt-source-latest-${release}.deb"
+    apt-get -y install "./groonga-apt-source-latest-${release}.deb"
+    rm "./groonga-apt-source-latest-${release}.deb"
+    touch "$STAMP_FILE"
 elif [[ "$release" =~ ^(buster)$ ]]; then
     apt-key add "$SCRIPTS_PATH"/setup/pgdg.asc
-    apt-key add "$SCRIPTS_PATH"/setup/pgroonga-debian.asc
     cat >$SOURCES_FILE <<EOF
 deb http://apt.postgresql.org/pub/repos/apt/ $release-pgdg main
 deb-src http://apt.postgresql.org/pub/repos/apt/ $release-pgdg main
-
-deb https://packages.groonga.org/debian/ $release main
-deb-src https://packages.groonga.org/debian/ $release main
 EOF
+    if ! dpkg-query -l | grep -q groogna-apt-source; then
```

Comment:
`dpkg-query -l` also list removed packages. You need something like `dpkg-query --showformat '${Status}\n' -W groonga-apt-source | grep -q ' installed$'`.

Reply:
Good catch!
I changed this to `! dpkg-query -l | grep -q "^ii  groogna-apt-source"` beucase `dpkg-query -W groonga-apt-source` shows `dpkg-query: no packages found matching groonga-apt-source` warning message when `groonga-apt-source` doesn't exist.